content/en/docs/install-operate/source.md

```diff
@@ -278,35 +282,8 @@ You can notice this observing that the option `USE_BUNDLED_DEPS` is `OFF` by def
 
 Changing such option to `ON` causes Falco build to bundle all the dependencies statically.
 
-For the sake of completeness this is the complete list of Falco dependencies:
```

Comment:
This is removed because it's hard to keep up to date and it does not provide much value here; I would prefer we had a better dependency management process upstream instead.